eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -30,7 +30,7 @@ impl Eth2NearRelay {
             eth1_rpc_client: Eth1RPCClient::new(&config.eth1_endpoint),
             eth_client_contract: EthClientContract::new(contract_wrapper),
             max_submitted_headers: config.total_submit_headers as u64,
-            current_gap_between_finalized_and_signature_slot: 96,
+            current_gap_between_finalized_and_signature_slot: 97,
```

Comment:
Let's make this value a constant which is constructed by having an appropriate explanation and values involved (e.g. epoch_size)

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/823867b1ba6f883c2ba735cd2f94c2910f26b069